nvflare/private/fed/server/job_runner.py

```diff
@@ -465,6 +466,7 @@ def run(self, fl_ctx: FLContext):
                             with self.lock:
                                 self.running_jobs[job_id] = ready_job
                             job_manager.set_status(ready_job.job_id, RunStatus.RUNNING, fl_ctx)
+                            self.log_info(fl_ctx, f"Job: {job_id} started to run, status changed to RUNNING.")
```

Comment:
I think this maybe not necessary. Once we have the new log, we can check if we have the following line "started to run, status changed to RUNNING". This will give us the information if the lock section has passed or not. We already have the schedule history update logging right before this.